_locales/_experimental_en.json

```diff
@@ -10,5 +10,29 @@
   },
   "hide_password": {
     "message": "Hide password"
+  },
+  "setting_profile": {
+    "message": "Profil"
+  },
+  "setting_profile_displayname": {
+    "message": "Displayname"
+  },
+  "setting_profile_status": {
+    "message": "Signature / Status"
+  },
```

Comment:
There should be translations in the normal android translations for this, no need to add own/experimental translations

Reply:
I updated the profile ones, but the other translations aren't pulled in from transifex yet.